snapcraft.yaml

```diff
@@ -342,6 +368,24 @@ parts:
       echo "ac_add_options --with-mozilla-api-keyfile=$SNAPCRAFT_STAGE/mozilla-desktop-geoloc-api.key" >> $MOZCONFIG
     override-prime: ''
 
+  debug-symbols:
+    plugin: nil
+    build-packages:
+      - python3
+      - python3-virtualenv
+    after:
+      - firefox
+    override-pull: |
+      if [ -f "$SNAPCRAFT_PROJECT_DIR/symbols-upload-token" ]; then
+        export SYMBOLS_ARCHIVE=$(find $SNAPCRAFT_STAGE/debug-symbols/ -type f -name "firefox-*.crashreporter-symbols.zip")
+        virtualenv venv/
+        source venv/bin/activate
+        venv/bin/pip3 install redo requests argparse
+        SOCORRO_SYMBOL_UPLOAD_URL=https://symbols.stage.mozaws.net/upload/ SOCORRO_SYMBOL_UPLOAD_TOKEN_FILE="$SNAPCRAFT_PROJECT_DIR/symbols-upload-token" venv/bin/python3 $SNAPCRAFT_STAGE/debug-symbols/upload_symbols.py $SYMBOLS_ARCHIVE
+        rm "$SNAPCRAFT_PROJECT_DIR/symbols-upload-token"
+        deactivate
```

Comment:
I guess we'll want to `else` and copy somewhere where `launchpad` can pull data for handling beta, stable and esr?